Reconstruction/RecCalorimeter/tests/options/runFullCaloSystem_ReconstructionSW_noNoise.py

```diff
@@ -3,16 +3,16 @@
 ecalBarrelCellsName = "ECalBarrelCells"
 ecalEndcapCellsName = "ECalEndcapCells"
 ecalFwdCellsName = "ECalFwdCells"
-hcalBarrelCellsName = "newHCalCells"
-hcalExtBarrelCellsName = "newExtHCalCells"
+hcalBarrelCellsName = "HCalBarrelCells"
+hcalExtBarrelCellsName = "HCalExtBarrelCells"
 hcalEndcapCellsName = "HCalEndcapCells"
 hcalFwdCellsName = "HCalFwdCells"
 # Readouts
 ecalBarrelReadoutName = "ECalBarrelPhiEta"
 ecalEndcapReadoutName = "EMECPhiEta"
 ecalFwdReadoutName = "EMFwdPhiEta"
-hcalBarrelReadoutName = "BarHCal_Readout_phieta"
-hcalExtBarrelReadoutName = "ExtBarHCal_Readout_phieta"
+hcalBarrelReadoutName = "HCalBarrelReadout"
```

Comment:
This readout does not contain eta-phi segmentation, it will not work in sliding window reconstruction.